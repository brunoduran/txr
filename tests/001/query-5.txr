@(bind cmd-rx #/sshd.*/)
@(collect)
@nil @nil @nil @nil @nil @nil @nil @{cmd cmd-rx}
@(end)
