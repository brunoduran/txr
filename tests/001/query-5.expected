cmd-rx="#/sshd.*/"
cmd[0]="sshd: kaz [priv] "
cmd[1]="sshd: kaz@pts/2  "
cmd[2]="sshd: kaz [priv] "
cmd[3]="sshd: kaz@pts/4  "
cmd[4]="sshd: kaz [priv] "
cmd[5]="sshd: kaz@pts/1  "
